utils/epinow2/functions.py

```diff
@@ -15,6 +15,7 @@ def extract_user_args() -> dict:
     state = os.environ.get("state", "all")
     disease = os.environ.get("disease", "all")
     report_date = os.environ.get("report_date", date.today())
+    production_date = os.environ.get("production_date", date.today())
```

Comment:
I set the default to be today; making it the Friday of the current week is kind of annoying, but if we feel strongly about it I can implement @zsusswein